src/main.cpp

```diff
@@ -79,17 +81,21 @@ void scani2c() {
     else
         Serial.println("done\n");
 }
+
 //! Arduino setup function
 void setup() {
-    // while (!Serial) {
-    //     ; // wait for serial port to connect. Needed for native USB
-    // }
+    Serial.begin(SERIAL_BAUD_RATE);
+    while (!Serial) {
+        ; // wait for serial port to connect. Needed for native USB
+    }
     Serial.println("Starting setup");
 
     while (nh.connected()) {
```

Comment:
Shouldn't this be `while(!nh.connected()) { ... }`

Reply:
Already there on line nh.connected(), removed the redundant troubleshooting lines.